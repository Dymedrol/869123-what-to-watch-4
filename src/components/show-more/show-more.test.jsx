import React from 'react';
import renderer from 'react-test-renderer';
import ShowMore from './show-more.jsx';

describe(`проверяем ShowMore`, () => {
  it(`проверяем снепшот ShowMore`, () => {
    const tree = renderer.create(
        <ShowMore
          allMoviesLength={20}
          filtredMoviesLength={5}
          onShowMoreClickHandler={() => {}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`проверяем снепшот ShowMore без кнопки`, () => {
    const tree = renderer.create(
        <ShowMore
          allMoviesLength={8}
          filtredMoviesLength={10}
          onShowMoreClickHandler={() => {}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
