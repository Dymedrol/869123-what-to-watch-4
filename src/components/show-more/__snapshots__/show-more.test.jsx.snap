// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`проверяем ShowMore проверяем снепшот ShowMore 1`] = `
<div
  className="catalog__more"
>
  <button
    className="catalog__button"
    onClick={[Function]}
    type="button"
  >
    Show more
  </button>
</div>
`;

exports[`проверяем ShowMore проверяем снепшот ShowMore без кнопки 1`] = `
<div
  className="catalog__more"
/>
`;
